src/V3AstNodes.h

```diff
@@ -2427,6 +2427,37 @@ class AstArg : public AstNode {
     bool emptyConnectNoNext() const { return !exprp() && name() == "" && !nextp(); }
 };
 
+class AstActiveRegion : public AstNodeRegion {
```

Comment:
Remains to be seen... but I suspect we don't need a different node class for each kind of region, a single "AstRegion" with the type inside it would do.